### PR TITLE
Update/csi 2300 align ug rm csv yamls

### PR DESCRIPTION
**Pull request by ArbelNathan**

### CODE REVIEW

**Review by oriyarde — APPROVED** [review]

**Review by barakda — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,17 +4,15 @@ The Container Storage Interface (CSI) Driver for IBM block storage systems enabl
...
-  - OpenShift v4.5 (x86)
-  - Kubernetes v1.17 (x86)
+  - OpenShift v4.4 (x86, IBM Z, and IBM PowerPC)
+  - OpenShift v4.5 (x86, IBM Z, and IBM PowerPC)
   - Kubernetes v1.18 (x86)
+  - Kubernetes v1.19 (x86)
 
 Supported IBM storage systems:
   - IBM Spectrum Virtualize Family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R)
   - IBM FlashSystem A9000/R
```
Comment (rivkap) [inline README.md:14]: Can this be changed to IBM FlashSystem A9000 and A9000R ?

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -122,17 +122,15 @@ spec:
...
-      - OpenShift v4.5 (x86)
-      - Kubernetes v1.17 (x86)
+      - OpenShift v4.4 (x86, IBM Z, and IBM PowerPC)
+      - OpenShift v4.5 (x86, IBM Z, and IBM PowerPC)
       - Kubernetes v1.18 (x86)
+      - Kubernetes v1.19 (x86)
 
     Supported IBM storage systems:
       - IBM Spectrum Virtualize Family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (FlashSystem 5010, 5030, 5100, 7200, 9100, 9200, 9200R)
       - IBM FlashSystem A9000/R
```
Comment (rivkap) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml:132]: IBM FlashSystem A9000 and A9000R

**Review by rivkap — APPROVED** [review]
Approving but if possible, to change the two places mentioning FlashSystem A9K

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml (hunk 2 above)
Comment (ArbelNathan) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.4.0/ibm-block-csi-operator.v1.4.0.clusterserviceversion.yaml]: done

File: README.md (hunk 1 above)
Comment (ArbelNathan) [reply README.md]: done

**Pull request merged**